package io.wiem.exceptions;

public class UserNotFoundException extends Throwable {
}
